src/commands/cork_proposal/cellar_v2_2/add_adaptor_to_catalogue.rs

```diff
@@ -0,0 +1,72 @@
+use abscissa_core::{clap::Parser, Command, Runnable};
+
+use crate::{
+    application::APP,
+    cellars::{self, is_evm_address},
+    commands::cork_proposal::print_proposal,
+    prelude::*,
+    proto::{
+        cellar_v2_2governance::{AddAdaptorToCatalogue, Function},
+        governance_call::Call,
+        CellarV22governance, GovernanceCall,
+    },
+};
+
+#[derive(Command, Debug, Parser)]
+#[clap(
+    long_about = "DESCRIPTION\n\nCalls addAdaptorToCatalogue() on the target V2.2 cellar contract at the specified block height.\nFor more information see https://github.com/PeggyJV/cellar-contracts/blob/main/src/base/Cellar.sol"
+)]
+pub struct AddAdaptorToCatalogueCmd {
+    #[clap(short = 'a', long)]
```

Comment:
Any reason why we call out a specific `short` value here but not for others?

Reply:
Nope. I think I copied some args structs from other commands and it was leftover.